backend/src/controllers/database.py

```diff
@@ -0,0 +1,152 @@
+from flask import Blueprint, make_response, request
+
+from src.database.crud.album import (
+    add_genres_to_album,
+    get_album_artists,
+    get_album_genres,
+    get_user_albums,
+    update_album,
+)
+from src.database.crud.genre import create_genre
+from src.database.crud.playlist import (
+    create_playlist,
+    get_playlist_albums,
+    get_playlist_by_id_or_none,
+    get_user_playlists,
+    update_playlist,
+)
+from src.database.crud.user import get_or_create_user
+from src.musicbrainz import MusicbrainzClient
+from src.spotify import SpotifyClient
+from time import sleep
+
+
+def database_controller(spotify: SpotifyClient, musicbrainz: MusicbrainzClient):
```

Comment:
Very minor, but conceptually this name bugs me because it's very much exposing internal details as API details... but I can't immediately offer better! The conceptual tie between the things we own vs Spotify is strong

I'll leave this as a thought, but I wouldn't worry about changing it

Reply:
Yeah, it doesn't quite seem right. I think once I've got the database management happening independently I can get rid of it entirely. Will leave as is for now though.